module-7-web-apis/todo/src/localStorageHelper.js

```diff
@@ -0,0 +1,48 @@
+import {Todo} from './todo.js';
```

Comment:
I do not know where to comment my suggestions, so I put it here. :)
- Accessing localStorage directly so many times is not optimal in performance perspective.
I rather get all todo item from the localStorage when the app starts, and store it in a local variable. And update both the localStorage and local state when something happens on UI. 
- I rather store all of the todo-s in one record in the storage. (JSON.stringify, JSON.parse)

Reply:
Hmm. I thought about this beforehand, and i made the decision to keep todos separately in localStorage and to operate directly on localStorage. The reasons behind that are:

- if we would have a local variable and the localStorage and lets say that we are updating both when an action happens, we ended up updating the local variable and the localStorage. So basically we have more operations to do.
- If we would store all of the todo's in one record, we would be in need to override the whole list of todos just because one of the todo's has been changed.

Not sure are the above reasons make sense or not. It made sense when I started to work on this task :D. 
Let me know! Thanks!